corehq/apps/app_manager/tests/test_suite.py

```diff
@@ -1033,6 +1033,48 @@ def _load_case(self, child_module_form, case_type, parent_module=None):
             module.parent_select.module_id = parent_module.unique_id
 
 
+class UserCaseOnlyModuleAsChildTest(BasicModuleAsChildTest):
+    """
+    Even though a module might be usercase-only, if it acts as a parent module
+    then the user should still be prompted for a case of the parent module's
+    case type.
+
+    The rationale is that child cases of the usercase never need to be
+    filtered by a parent module, because they can't be filtered any more than
+    they already are; there is only one usercase.
+    """
+
+    def setUp(self):
+        super(UserCaseOnlyModuleAsChildTest, self).setUp()
+        self.is_usercase_in_use_mock.return_value = True
+
+    def test_child_module_session_datums_added(self):
+        self.module_1.root_module_id = self.module_0.unique_id
+        self.module_0.case_type = 'gold-fish'
+        m0f0 = self.module_0.get_form(0)
+        # m0 is a user-case-only module. m0f0 does not update a normal case, only the user case.
+        m0f0.requires = 'case'
```

Comment:
This should only be set if you're loading a case. Removing this line will remove the `<datum id="case_id"` from the `m0-f0` entry.